### PR TITLE
docs: add link to EOL blog post and update guidance

### PR DESCRIPTION
**Pull request by Splaktar**
- update copyright year
- add more context around AngularJS Material and Angular Material (based on review feedback)
- add link to Angular core package (based on review feedback)

Related to https://github.com/angular/bower-angular/pull/49

cc: @mgol @mgechev @gkalpak @aaronfrost

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
-This repository publishes the AngularJS Material v1.x library and localized installs
-using `npm`. You can find the component source-code for this library in the
-[AngularJS Material repository](https://github.com/angular/material).
+# AngularJS Material
+
+**[Support for legacy AngularJS ended on January 1st, 2022](https://goo.gle/angularjs-end-of-life).**
```
Comment (mgechev) [inline README.md:3]: Can we have a sentence about Angular here?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
 
-> Please file issues and pull requests against that `angular/material` repository only. Do not file
-issues here on the publishing repository.
+See the following, related, actively supported Angular packages:
+- [`@angular/material`](https://www.npmjs.com/package/@angular/material)
+- [`@angular/cdk`](https://www.npmjs.com/package/@angular/cdk)
+
+This repository was used to publish the AngularJS Material v1.x library and localized installs
```
Comment (mgechev) [inline README.md:9]: Can we have a sentence about Angular Material here?

**Review by mgechev — CHANGES_REQUESTED** [review]
Few suggestions

File: README.md (hunk 2 above)
Comment (Splaktar) [reply README.md:10]: Added below.

File: README.md (hunk 1 above)
Comment (Splaktar) [reply README.md]: Added.

**Review by mgechev — APPROVED** [review]

**Pull request merged**